Randolph-dev/Druid_Partnering_Project_React24k
# SCRUM-60 Documentation: Core configuration, build tools, and dependen…

### Related ticket:
[SCRUM-60](https://druidpartneringproject2024bch.atlassian.net/browse/SCRUM-60)

### In this PR:
Initialized Mautic setup with standard configuration files.

Core files added:
- composer.json/lock: PHP dependencies for Mautic core
- package.json/lock: Node.js dependencies for frontend
- .lando.yml: Local development environment config
- .gitignore: Proper ignore patterns for Mautic files (do **NOT** modify, only add as usual)

### Testing instructions (please read carefully)
1. `git pull origin` to download configurations in your branch
2. cd Mautic
3. lando start
4. lando composer install
5. npm install
6. lando info to check available URL port (e.g. http://localhost:[port])
7. Run Mautic installation wizard
8. **IMPORTANT:** In the installation wizard, you **MUST** use these exact database settings:   
   - Database host: database
   - Database name: lamp
   - Database user: lamp
   - Database password: lamp
 
 (These values are required for Lando's database connection)
  
9. Create your own admin credentials. 

[SCRUM-60]: https://druidpartneringproject2024bch.atlassian.net/browse/SCRUM-60?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
